[quant][graphmode][fix] Don't permute in quantized::conv2d pattern

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #27403 [quant][graphmode] Minor fix and turn off fold_convbn
* #27348 [quant][graphmode] Add e2e test for conv+bn
* **#27347 [quant][graphmode][fix] Don't permute in quantized::conv2d pattern**

Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18182919](https://our.internmc.facebook.com/intern/diff/D18182919)

jerryzh168 committed:
[quant][graphmode][fix] Don't permute in quantized::conv2d pattern

Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

ZolotukhinM reviewed (approved):
It looks good, but don't we need to adjust some tests too?

jerryzh168 commented:
> It looks good, but don't we need to adjust some tests too?

I'll add a single conv test, we don't have that yet.

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18182919](https://our.internmc.facebook.com/intern/diff/D18182919)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18182919](https://our.internmc.facebook.com/intern/diff/D18182919)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18182919](https://our.internmc.facebook.com/intern/diff/D18182919)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18182919](https://our.internmc.facebook.com/intern/diff/D18182919)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18182919](https://our.internmc.facebook.com/intern/diff/D18182919)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] Don't permute in quantized::conv2d pattern"


Summary:
it's already done in the op, we don't need to permute again

Test Plan:
test_jit.py
we'll test in e2e tests

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18182919](https://our.internmc.facebook.com/intern/diff/D18182919)

[ghstack-poisoned]
